dair_pll/multibody_learnable_system.py

```diff
@@ -114,20 +115,32 @@ def get_visualization_system(self) -> DrakeSystem:
         :py:class:`MultibodyLearnableSystem` has a duplicate, and visualization
         elements are repainted for visual distinction.
 
+        Args:
+            new_geometry: Whether or not to use learned geometry.
+
         Returns:
             New :py:class:`DrakeSystem` with doubled state and repainted
             elements.
         """
-        if not self.visualization_system:
+        if new_geometry or not self.visualization_system:
```

Comment:
Reason for `new_geometry` -> regenerate visualization system is non obvious enough that I think it deserves a comment